MyfansDownloader_unified.py

```
@@ -0,0 +1,125 @@
+        }
+        
+        print("Choose an option:")
+        for key, (desc, _) in options.items():
+            print(f"{key}. {desc}")
+        
+        choice = input("\nEnter your choice (1 or 2): ").strip()
+        
+        if choice in options:
+            _, script_path = options[choice]
+            print(f"\nRunning {options[choice][0]}...\n")
+            subprocess.run([sys.executable, script_path], check=False)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)